src/Accordion/index.js

```diff
@@ -105,7 +105,6 @@ export default class Accordion extends Component {
     return (
       <Root
         className={cx('react-sanfona', className)}
-        role="tablist"
```

Comment:
My understanding is that a 'tablist' should appear inside a 'tabpanel', and this element doesn't.

Also, I don't think an Accordion is really a 'tab'; I note that the ARIA 'best practise' implementation does not have this role set?

https://www.w3.org/TR/wai-aria-practices-1.1/#accordion